### PR TITLE
docs: don't allow patches for linux formula either

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

**Review by dawidd6 — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
👍🏻 🙇🏻

**Review by Moiss27 — CHANGES_REQUESTED** [review]
i need to be closed out of this thread, i dont know anything abouy this but i know its the root of all my computers being hacked. im not asking again, delete my profile

File: docs/Acceptable-Formulae.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ due to upstream changes and we can’t provide [bottles](Bottles.md) for them.
 ### Niche (or self-submitted) stuff
 The software in question must:
 
-* be maintained (i.e. the last release wasn't ages ago, it works without patching on all supported macOS releases and has no outstanding, unpatched security vulnerabilities)
+* be maintained (i.e. the last release wasn't ages ago, it works without patching on all Homebrews supported OS versions and has no outstanding, unpatched security vulnerabilities)
```
Comment (nandahkrishna) [inline docs/Acceptable-Formulae.md:43]:
```suggestion
* be maintained (i.e. the last release wasn't ages ago, it works without patching on all OS versions supported by Homebrew and has no outstanding, unpatched security vulnerabilities)
```

Comment (carlocab) [reply docs/Acceptable-Formulae.md:43]: Or:
```suggestion
* be maintained (i.e. the last release wasn't ages ago, it works without patching on all Homebrew-supported OS versions and has no outstanding, unpatched security vulnerabilities)
```

**Review by nandahkrishna — APPROVED** [review]

**Pull request merged**